### DISCUSSION TITLE
USB Wireless Adapter/OS

### DISCUSSION THREAD

**Discussion by salibeh** [ZerBea/hcxdumptool#476] [General]
Hi,
I am looking for a USB Wireless Adapter that is capable of monitoring/Capturing WDS (4-Address, Wireless Distribution System) 802.11  Frames in the 5.0 GHz band.   Also, the recommended Linux OS (Kali, Ubuntu, Debian, etc.) and kernel releases that supports such a device for the most reliable results.  Please suggest other venues tfor me o make this request if this is not the right venue
Thanks,
Emil

**Comment by ZerBea** [comment]
The recommendation for an operating system is easy:
```
Rank 1: OpenWRT compiled via toolchain
Rank 2: Arch Linux (systemd-boot,  linux and linux-lts)
``` 

The recommendation for an operating system is not easy, because it depends on the target. E.g. if you're targeting an AC network, you need an AC adapter.
As of today I'm using an ALFA AWUS036ACHM, an ALFA AWUS036ACM, an ALFA AWUS036NHA (2.4 only) and an ALFA 
AWUS036NH (2.4 only) to test and improve hcxdumptool (targeting WPA PSK and WPAPSK256 only).

AWUS036ACHM and ALFA AWUS036ACM support several modes:
```
	Supported interface modes:
		 * IBSS
		 * managed
		 * AP
		 * AP/VLAN
		 * monitor
		 * mesh point
		 * P2P-client
		 * P2P-GO
```
and a wide range of frequencies:
```
available frequencies: frequency [channel] tx-power of Regulatory Domain: DE
  2412 [  1] 20.0 dBm	  2417 [  2] 20.0 dBm	  2422 [  3] 20.0 dBm	  2427 [  4] 20.0 dBm
  2432 [  5] 20.0 dBm	  2437 [  6] 20.0 dBm	  2442 [  7] 20.0 dBm	  2447 [  8] 20.0 dBm
  2452 [  9] 20.0 dBm	  2457 [ 10] 20.0 dBm	  2462 [ 11] 20.0 dBm	  2467 [ 12] 20.0 dBm
  2472 [ 13] 20.0 dBm	  2484 [ 14] disabled	  5180 [ 36] 20.0 dBm	  5200 [ 40] 20.0 dBm
  5220 [ 44] 20.0 dBm	  5240 [ 48] 20.0 dBm	  5260 [ 52] 20.0 dBm	  5280 [ 56] 20.0 dBm
  5300 [ 60] 20.0 dBm	  5320 [ 64] 20.0 dBm	  5500 [100] 20.0 dBm	  5520 [104] 20.0 dBm
  5540 [108] 20.0 dBm	  5560 [112] 20.0 dBm	  5580 [116] 20.0 dBm	  5600 [120] 20.0 dBm
  5620 [124] 20.0 dBm	  5640 [128] 20.0 dBm	  5660 [132] 20.0 dBm	  5680 [136] 20.0 dBm
  5700 [140] 20.0 dBm	  5720 [144] 13.0 dBm	  5745 [149] 13.0 dBm	  5765 [153] 13.0 dBm
  5785 [157] 13.0 dBm	  5805 [161] 13.0 dBm	  5825 [165] 13.0 dBm	  5845 [169] 13.0 dBm
  5865 [173] 13.0 dBm	  5885 [177] disabled
```
I don't know how this adapters are performing when running against WDS networks, but a good place to get more/additional information is here:
https://github.com/morrownr/USB-WiFi/discussions

**Comment by ZerBea** [comment]
Moved this to discussions, because it is not an issue.
